assets/prototype/application/ui/components/display/entityListFilterBar/useEntityListFilterState.js

```diff
@@ -0,0 +1,89 @@
+import { useReducer, useEffect } from '@wordpress/element';
+
+const useEntityListFilterState = () => {
+	const initialState = {
+		perPage: 6,
+		searchText: '',
+		showEntityFilters: false,
+		view: 'list',
```

Comment:
let's change the default view to "grid" plz

```suggestion
		view: 'grid',
```